orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -46,13 +46,22 @@ RANGERGROUP="$5"
 CLOSECONNECTIONS="$6"
 DATABASENAMES="${@: 7}"
 
+# Root directory for local postgres dump.
+BACKUPS_DIR="/var/tmp/"
+# Directory for the current postgres dump.
+DATE_DIR=${BACKUPS_DIR}/$(date '+%Y-%m-%dT%H:%M:%SZ')
+
 {%- from 'postgresql/settings.sls' import postgresql with context %}
 {% if postgresql.ssl_enabled == True %}
 export PGSSLROOTCERT="{{ postgresql.root_certs_file }}"
 export PGSSLMODE=verify-full
 {%- endif %}
 
 errorExit() {
+  if [ -d "$DATE_DIR" ]; then
+    rm -rf -v "$DATE_DIR" > >(tee -a $LOGFILE) 2> >(tee -a $LOGFILE >&2)
```

Comment:
What about already existing temp dirs? Those should be cleaned up right?

Reply:
Thank you for the review! I'm not sure what are those existing temp dirs? This command will remove all the files under `/var/tmp/`. Are there any other tmp dirs that exist somewhere else?

Reply:
@lnardai I got what you meant! Do you mean the old date directories? That is a good point!! [This]([CDPSDX-3767](https://jira.cloudera.com/browse/CDPSDX-3767)) jira's purpose is to remove the temp files left by this failed backup and it will make sure we no longer have this issue moving forward. Moving directories not related to the current backup is a more risky step I think since we cant be 100% sure if the customer needs them. I can create a new jira to look into that if needed!

Reply:
Hi @lnardai after talking with my manager, we created this jira https://jira.cloudera.com/browse/CDPSDX-3791 to look into that issue. Thank you!!